packages/docs/content/migration-guides/standardize-theme-colors.mdx

```diff
@@ -89,7 +87,7 @@ One major shift with color standardization is updating the `secondary` color to
 
 
 <Alert variation="warn" weight="lightweight" hideIcon="true" className="ds-u-measure--wide">
-  <b>What does this mean:</b> If you are using <code>secondary</code> colors in your design,
+  <b>What to do:</b> If you are using <code>secondary</code> colors in your design,
```

Comment:
For all of these 'what to do': This makes it seem like it's a necessary change, where the text following offers a question. 'If you are...' which could possibly be a little confusing. Perhaps `Note:` or `Changed:` or even just getting rid of it entirely and leaving the question? I dunno.

Reply:
Yea that's a good point. The idea is that those `what to do` sections would be like a TLDR if you are to read or do one thing in this section make it this. The intent was to also make it feel actionable for product teams. 

I'd have concerns with `note:` or `changed:` that it would be glossed over... 

Curious about others thoughts too @pwolfert and @brentschneider